src/schema/v2/location.ts

```diff
@@ -106,6 +107,22 @@ export const LocationType = new GraphQLObjectType<any, ResolverContext>({
 
     display: {
       type: GraphQLString,
+      resolve: ({
+        display,
+        address,
+        address_2,
+        city,
+        state,
+        postal_code,
+        country,
+      }) => {
+        return (
+          display ||
+          compact([address, address_2, city, state, postal_code, country]).join(
+            ", "
+          )
```

Comment:
**issue (non-blocking):** I was taught to not place a comma between the state and postal code when addressing letters as a kid, so the resulting address string will look funny to me, but I don't think it matters.